README.md

```diff
@@ -101,6 +101,20 @@ echo ">> Deploy Gateway API resources"
 kubectl apply -f ./third_party/contour
 ```
 
+#### Envoy Gateway
+
+```bash
+echo ">> Bringing up Envoy Gateway"
+helm repo add envoyproxy https://charts.envoyproxy.io
+helm install eg oci://docker.io/envoyproxy/gateway-helm \
+  --version ${ENVOY_GATEWAY_VERSION} \
+  -n envoy-gateway-system --create-namespace \
+  -f ./third_party/envoy-gateway/helm/values-eg.yaml
+
+echo ">> Deploy Gateway API resources"
+kubectl apply -f ./third_party/envoy-gateway
```

Comment:
```suggestion
kubectl apply --server-side -f ./third_party/envoy-gateway
```

With incoming 1.4 it seems like we need this :/

Reply:
Oh, Thank you 🙏